libs/service-portal/settings/access-control/src/screens/Access/components/AccessItem/AccessItem.tsx

```diff
@@ -117,11 +118,12 @@ function AccessItem({ apiScopes, authDelegation }: PropTypes) {
                 ...tdStyling,
                 paddingLeft: isFirstItem ? 3 : 8,
               }}
+              style={tableStyles}
             >
               <CheckboxController
                 id={`${item.model}.name`}
                 spacing={0}
-                labelVariant={isFirstItem ? 'default' : 'small'}
+                labelVariant={isFirstItem ? 'default' : 'medium'}
```

Comment:
I can't find these pages any more in the navigation:

![image](https://user-images.githubusercontent.com/115094/146917121-123325f5-578f-43dd-8503-59490afd1a0f.png)

Reply:
Yes correct, it's now only available under UserMenu

Reply:
Interesting 👍